src/coreclr/jit/emit.cpp

```diff
@@ -4129,42 +4129,15 @@ void emitter::emitDispCommentForHandle(size_t handle, size_t cookie, GenTreeFlag
 
     const char* str = nullptr;
     if (flag == GTF_ICON_STR_HDL)
+    {
+        str = "string handle";
```

Comment:
I don't print comment for `GTF_ICON_STR_HDL` because it's expected to be rare now (string literals are mostly frozen) and didn't want to keep `eeGetCPString` for it because that is a bit dangerous and contained a gc hole

Reply:
I don't think there was a GC hole since the users of `eeGetCPString` do not call any JIT-EE functions between reading the handle and reading the characters, so no GC could occur.

Reply:
I do not think that this runs in coop mode. JITing is a long running piece of unmanaged code. It always runs in preemptive mode to avoid blocking GC.

Reply:
Ah okay, so there is a GC hole. Sorry for the misinformation.